<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreseedDevice</class>
 <widget class="QWidget" name="PreseedDevice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #17394A</string>
  </property>
  <widget class="QPushButton" name="installoptionsnextButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>200</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(240, 240, 240)</string>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="installoptionsdescriptionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>561</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(240, 240, 240)</string>
   </property>
   <property name="text">
    <string>where would you like to install OSMC?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="installoptionsLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>561</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(240, 240, 240)</string>
   </property>
   <property name="text">
    <string>install options</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="radiobuttongroupBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(240, 240, 240)</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="nfsinstallradioButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Souce Pro</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(240, 240, 240)</string>
    </property>
    <property name="text">
     <string>on an NFS share</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="usbinstallradioButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Souce Pro</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(240, 240, 240)</string>
    </property>
    <property name="text">
     <string>on a USB stick</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sdinstallradioButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Souce Pro</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(240, 240, 240)</string>
    </property>
    <property name="text">
     <string>on an SD card</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="emmcinstallradioButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Souce Pro</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(240, 240, 240)</string>
    </property>
    <property name="text">
     <string>on internal storage</string>
    </property>
   </widget>
   <zorder>nfsinstallradioButton</zorder>
   <zorder>usbinstallradioButton</zorder>
   <zorder>sdinstallradioButton</zorder>
   <zorder>nfsinstallradioButton</zorder>
   <zorder>usbinstallradioButton</zorder>
   <zorder>sdinstallradioButton</zorder>
   <zorder>emmcinstallradioButton</zorder>
  </widget>
  <zorder>radiobuttongroupBox</zorder>
  <zorder>installoptionsnextButton</zorder>
  <zorder>installoptionsdescriptionLabel</zorder>
  <zorder>installoptionsLabel</zorder>
  <zorder>emmcinstallradioButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
